examples/hitl/rearrange/rearrange.py

```diff
@@ -4,28 +4,49 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+
+# temp until hitl framework is a proper package
+def add_hitl_framework_import_path():
+    import os
+    import sys
+
+    current_script_directory = os.path.dirname(os.path.realpath(__file__))
+    parent_directory = os.path.abspath(
+        os.path.join(current_script_directory, "../../siro_sandbox/")
+    )
+    sys.path.append(parent_directory)
+
+
+add_hitl_framework_import_path()
+
+import hitl_main
 import magnum as mn
 import numpy as np
 from app_states.app_state_abc import AppState
+from app_states.app_state_tutorial import AppStateTutorial
 from camera_helper import CameraHelper
 from controllers.gui_controller import GuiHumanoidController
 from gui_navigation_helper import GuiNavigationHelper
+from hitl_arg_parser import create_hitl_arg_parser
+
+# from hydra_helper import register_hydra_plugins  # coming soon
 from utils.gui.gui_input import GuiInput
 from utils.gui.text_drawer import TextOnScreenAlignment
 from utils.hablab_utils import (
     get_agent_art_obj_transform,
     get_grasped_objects_idxs,
 )
 
+import habitat_baselines
+
 
 class AppStateRearrange(AppState):
```

Comment:
If we have time, it will be useful to describe the various things that happen in the rearrange app. Basically, since we have many functions without much comments, adding some comments might help getting a quick, good view on various things that are happening in the app. Just a nit again.